src/App.tsx

```diff
@@ -23,28 +23,44 @@ import ExplorePage from "./modules/ExplorePage/Components/ExplorePage/ExplorePag
 import DetailsPage from "./modules/DetailsPage/Components/DetailsPage/DetailsPage";
 import Favorites from "./modules/Favorites/Components/Favorites/Favorites";
 import AdvertisementsList from "./modules/Advertisements/Components/AdvertisementsList/AdvertisementsList";
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { CssBaseline } from "@mui/material";
+import BookingList from "./modules/Booking/Components/Booking/BookingList";
 
 function App() {
+
+  const theme = createTheme({
+    typography: {
+      fontFamily: [
+  
+        'Poppins',
+       
+      ].join(','),
+    },
+  });
+    
   const routes = createBrowserRouter([
     {
       path: "",
       element: <AuthLayout />,
       errorElement: <NotFound />,
       children: [
+        { index: true, element: <Login /> },
+
         { path: "login", element: <Login /> },
+        { path: "", element: <Login /> },
         { path: "register", element: <Registeration /> },
         { path: "verify-user", element: <Verify /> },
         { path: "forget-password", element: <ForgetPassword /> },
         { path: "reset-password", element: <ResetPassword /> },
-        { path: "change-password", element: <ChangePassword /> },
+        { path: "change-password", element: <ProtectedRoute><ChangePassword /></ProtectedRoute> },
       ],
     },
     {
       path: "",
       element: <LandingPageLayout />,
```

Comment:
The user section with the Landing page should be first. When entering the website, the landing page should be the first page that we see unless we are an admin user.